App/theme.ts

```diff
@@ -112,7 +129,7 @@ export const Buttons = StyleSheet.create({
   primaryDisabled: {
     padding: 16,
     borderRadius: 4,
-    backgroundColor: 'rgba(66, 128, 62, 0.3)',
+    backgroundColor: Colors.darkGreenHeavyTransparent,
```

Comment:
Should we avoid using color names in the theme? When we style to the BC Pallet will it make things more complicated (we don't have a green).

Reply:
I went back and forth on this. My thoughts were to keep things DRY and avoid situations where we would have to make updates to multiple values. Instead of naming colours is it better for us to stick to naming conventions more commonly used in style guides (ex primary, secondary, info, error, success, warning, etc). It would also be nice to specify colour shades and opacity without having to use names like the one referenced here.

Reply:
Can we address this with theming?

Reply:
Yup.